package.json

```diff
@@ -60,7 +60,7 @@
     "eslint-plugin-import": "^2.23.2",
     "flow-bin": "^0.151.0",
     "inquirer": "^8.1.0",
-    "jest": "27.0.6",
```

Comment:
You are correct that this was just an npm update thing...not anything I did on purpose. So we can ignore it.